src/rust/ensogl/src/display/shape/text/text_field/frp/keyboard.rs

```diff
@@ -167,6 +168,57 @@ impl TextFieldKeyboardFrp {
 }
 
 
+// === Keys combinations ===
```

Comment:
These things are very hacky but for now they are ok. Just a note that using the new shortcut manager it should be done way nicer.